src/lib/hash.ts

```diff
@@ -78,3 +86,70 @@ function prefixToField(prefix: string) {
     .flat();
   return Field.ofBits(bits);
 }
+
+/**
+ * Convert the {fields, packed} hash input representation to a list of field elements
+ * Random_oracle_input.Chunked.pack_to_fields
+ */
+function packToFields({ fields = [], packed = [] }: Input) {
```

Comment:
this is the last step that's necessary to convert the result of `toInput` to something which can be hashed